TerrorConsole/Assets/_Main/Scripts/Audio/AudioData.cs

```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
+
+namespace TerrorConsole
+{
+    [CreateAssetMenu(fileName = "AudioData", menuName = "ScriptableObjects/AudioData")]
+
+    public class AudioData : ScriptableObject
+    {
+        public string audioName;
+        public AudioClip audioClip;
+        public float volume = 1;
```

Comment:
Since these are a Data Structure makes sense they are "public" but that also means these should have the first character uppercase.